samples/deployment/fabric-smart-client/the-simple-testing-network/README.md

```diff
@@ -0,0 +1,88 @@
+# The simple testing network
+
+This gives an example how to run your FPC Chaincode with a simple testing network build with the Fabric Smart Client.
+The network is specified in [pkg/topology/fabric.go](pkg/topology/fabric.go).
+
+In this demo we use the demo fpc-echo chaincode.
+
+## Fabric network
+
+### Prep
+
+Let's make sure we have built the Enclave Registry Container Images and a FPC Chaincode Container Image.
+In this example we will run the [echo sample](../../../chaincode/echo). 
+
+Build ERCC image
+```bash
+make -C $FPC_PATH/ercc all docker
+```
+
+Build FPC-Echo image
+```bash
+export CC_NAME=echo
+export CC_PATH=$FPC_PATH/samples/chaincode/echo
+make -C $CC_PATH
+make -C $FPC_PATH/ecc DOCKER_IMAGE=fpc/fpc-${CC_NAME}$${HW_EXTENSION} DOCKER_ENCLAVE_SO_PATH=$CC_PATH/_build/lib all docker
+``` 
+
+Download fabric binaries:
+
+```bash
+curl -sSL https://bit.ly/2ysbOFE | bash -s -- 2.2.7 1.4.9 -s -d
+export FAB_BINS=$(PWD)/bin
```

Comment:
In the Fpc-development container, $(pwd) had to be lower case.
`export FAB_BINS=$(pwd)/bin`

Reply:
thanks! good catch!